proto/proxy.pb.go

```diff
@@ -36,7 +37,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
@cheftako @anfernee: I noticed that this was changed from `proto.ProtoPackageIsVersion2` to `proto.ProtoPackageIsVersion3` after removing vendor. Is there any implication of this I should be aware of? Still able to run proxy server and agent without any issue.

Reply:
Probably we move to the new proto version. I think it should be fine as long as it's consistent between agent and server

Reply:
Better to do it now while we are alpha